scripts/jenkins/track-upstream-and-package.pl

```diff
@@ -95,7 +95,8 @@ ()
   my $OSC_BUILD_DIST = $ENV{OSC_BUILD_DIST};
   unless($OSC_BUILD_DIST) {
     my $prj = `cat .osc/_project`;
-    $OSC_BUILD_DIST = "SLE_12_SP1";
+    $OSC_BUILD_DIST = "SLE_12_SP2";
+    $OSC_BUILD_DIST = "SLE_12_SP1" if ($prj =~ /Devel:Cloud:6/);
     $OSC_BUILD_DIST = "SLE_11_SP3" if ($prj =~ /Devel:Cloud:[1-5]/);
```

Comment:
Approved, but maybe a switch case statement makes more sense here. What do you think @dirkmueller ?